src/main/scala/tech/cryptonomic/conseil/tezos/DatabaseConversions.scala

```diff
@@ -176,6 +188,7 @@ object DatabaseConversions extends LazyLogging {
       val header = from.data.header
       val metadata = discardGenesis.lift(from.data.metadata)
       val CurrentVotes(expectedQuorum, proposal) = from.votes
+      val (year, month, day, time) = extractDateTime(toSql(header.timestamp))
```

Comment:
I guess we don't have any guarantee on how the json timestamp will be formatted nah? 🤔 
Otherwise we could simply parse the string...

Reply:
timestamp in tezos blocks has Z in the end, which stands for Zulu time zone, which is equal to UTC, which means its easily converted to timestamp in millis - had to do some investigation beforehand 😉

Reply:
Do you mean here that the timestamp always refers to Zulu, or that we could use a parser here, instead of converting to sql first?

Reply:
I don't think we have a original string  here anymore after parsing it as `ZonedDateTime`, so  converting it to timestamp gives me simple UTC time